### PR TITLE
Added PureMVC Standard (v1.0a) and Multicore (v1.1) repositories.

### PR DESCRIPTION
**Pull request by sparkbuzz**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -304,7 +306,7 @@
   "theseus": "npm:theseus",
   "theseus-angular": "npm:theseus-angular",
   "topo": "github:capaj/topo-browser",
-  "timers": "github:jspm/nodelibs-timers",
+  "timers": "github:jspm/nodelibs-timers",c
```
Comment (guybedford) [inline registry.json]: Can you remove this character change? I think that may be invalidating the JSON.

Comment (sparkbuzz) [inline registry.json]: Thanks for pointing out @guybedford , I've removed the char, must have entered it by mistake.

File: registry.json [hunk 2]
```diff
@@ -120,7 +120,7 @@
   "chai": "npm:chai",
   "chart": "github:nnnick/Chart.js",
   "child_process": "github:jspm/nodelibs-child_process",
-  "chosen": "github:harvesthq/chosen",
+  "chosen": "github:harvesthq/chosen",p
```
Comment (guybedford) [inline registry.json]: There's another character here as well!

**Comment by guybedford** [comment]
Thanks, looks like there's one more though.

**Comment by sparkbuzz** [comment]
Sorted my issues out, once merged, I'll retest to see if PureMVC installs correctly.

**Pull request merged**